packages/ui-view/src/View/README.md

```diff
@@ -607,61 +774,126 @@ class FocusedExample extends React.Component {
             borderRadius="none large"
             focusColor="danger"
             position="relative"
-            withFocusOutline={this.state.isFocused}
-            focusPosition={this.state.inset ? 'inset' : 'offset'}
+            withFocusOutline={isFocused}
+            focusPosition={inset ? 'inset' : 'offset'}
           >
             none large
           </View>
         </View>
       </View>
     )
   }
-}
 
-render(<FocusedExample />)
-```
+  render(<FocusedExample />)
+  ```
 
 Handle vertical and horizontal content overflow with the `overflowX` and `overflowY`
 props.
 
 > **Important CSS note:** Setting one axis to `visible` and setting the other to a different value results in `visible` behaving as `auto`.
 
-```javascript
----
-type: example
----
-class OverflowExample extends React.Component {
-  constructor (props) {
-    super(props)
+- ```javascript
+  class OverflowExample extends React.Component {
+    constructor(props) {
+      super(props)
 
-    this.state = {
-      overflowY: 'visible'
+      this.state = {
+        overflowY: 'visible'
+      }
     }
```

Comment:
Here only overFlowY is declared, but overFlowX is missing. Could you please add overFlowX too?

Reply:
Done